DataFormats/L1THGCal/interface/HGCalCluster.h

```diff
@@ -1,68 +1,95 @@
 #ifndef DataFormats_L1Trigger_HGCalCluster_h
```

Comment:
It's seems that all the indentations in this file have been changed. It makes the diff much more difficult to see. Please keep the indentation of the original author.